CustomColors.cpp

```diff
@@ -318,3 +322,170 @@ HOOK_METHOD_PRIORITY(FTLButton, OnRender, 5000, () -> void)
 
     g_isFTLButton = false;
 }
+
+
+// ---Custom Choice Colors---
+
+GL_Color COLOR_CHOICE_YELLOW = GL_Color(0.9529412f, 1.f, 0.3137255f, 1.f);
+GL_Color COLOR_CHOICE_GRAY = GL_Color(0.5882353f, 0.5882353f, 0.5882353f, 1.f);
+GL_Color COLOR_CHOICE_CYAN = GL_Color(0.f, 0.7647059f, 1.f, 1.f);
+
+std::unordered_map<std::string, ChoiceColor*> ChoiceColorMap;
+
+void ParseChoiceColorNode(rapidxml::xml_node<char>* node)
+{
+    if (node->first_attribute("name") == nullptr) return;
+
+    std::string name = std::string(node->first_attribute("name")->value());
+    if (ChoiceColorMap[name] != nullptr) return;
+
+    ChoiceColor* choiceColor = new ChoiceColor;
+    if(node->first_node("normal") != nullptr)
```

Comment:
There should be a space between the `if` and the opening parenthesis. You also shouldn't need to check `!= nullptr`, `node->first_node("normal")` will evaluate as `true` on its own if it's not null. All instances of these two issues should be fixed.